src/Model/PaymentMethod.php

```diff
@@ -90,6 +96,30 @@ public function setLast4(string $last4): self
         return $this;
     }
 
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    public function setName(string $name): self
+    {
+        $this->name = $name;
+
+        return $this;
+    }
+
+    public function getType(): string
+    {
+        return $this->type;
+    }
+
+    public function setType(string $type): self
```

Comment:
~I guess same enum question here...~

~EDIT: the other comment was on the other type.... should we make this an enum~

EDIT 2: I checked the docs & they are just returning it as a string... so :nothing-to-do-here: